### ISSUE TITLE
Display avatars

### ISSUE THREAD

**Issue by jayvdb** [manu-chroma/username-availability-checker#44]
https://github.com/manu-chroma/username-availability-checker/issues/35 added avatars to the backend.

We now need to display them.  Some UI design needs to be done first.

**Comment by manu-chroma** [comment]
So now that we're getting link to the profile picture, we can have two options for design: 

1. Divide ``status.html`` into three rows: 
	- Taken (with profile pic)
	- Taken (with no profile pic) eg: Behnance
	- Available user

2. Merge Taken (with/without profile pic) into one section and have dummy profile avatars in place for the design to look more consistent

**Comment by manu-chroma** [comment]
@jayvdb, I'm assigning it to the guy who originally did the frontend part of the website. He'll do a much better job than I can :sweat_smile:

**Comment by manu-chroma** [comment]
closing for now, since avatars are now displayed on the frontend. 

future work can include improving the UI.
